### ISSUE TITLE
Bug with radiant backdoor

### ISSUE THREAD

**Issue by SpeedyGonzaless** [jeffhill/Dota2#1664]
### Description

Because of radiant mid T2 tower is too close to the radiant base, when dire creeps near radiant mid T2 tower - the backdoor on bot or top T3 or T4 towers doesn't work. And this works ONLY with radiant mid T2 tower. If creeps for example near radiant bottom T2 tower then backdoor works on mid and top T3 towers. This bug quite breaks balance.
You can watch the replay of the match (37:57 on timer) and see that dire creeps are only near radiant mid T2 tower, but backdoor doesn't work on T3 top tower.

### Example Match ID (and possibly Timestamp)

6643103422

### Screenshots

![image](https://user-images.githubusercontent.com/40829475/176942699-4ab00e31-e402-4488-885a-abbe371a5cc0.png)
![image](https://user-images.githubusercontent.com/40829475/176942748-b09c327a-dae9-4a46-9211-8bf9e0d4ba1e.png)

**Comment by SonodaHanami** [comment]
facts:
the backdoor ranges work individually
radiant mid T2 is close
radiant T3 backdoor range includes mid T2(position of picture 2 creeps)

so I think the backdoor works without bug, though this is a balance issue indeed.

**Comment by TheZett** [comment]
This is how the "Backdoor in the base" aura is visualised:

![Backdoor_Protection_radius](https://user-images.githubusercontent.com/19760152/177017738-5f4ac985-c587-4aba-b14c-a266667ef753.jpg)

If any creep walks into this area, the backdoor protection for the entire base (ancient, t3s, t4s, etc) gets disabled.

The T2 towers have their own backdoor protection auras, thus they are separately disabled, not all at once.

**Comment by SpeedyGonzaless** [comment]
> 

Yes, I understand that there is radius of T3 backdoor, but it seems quite imbalance and confusing that there could be only one situation when creeps are near T2 tower and backdoor doesn't work on T3 towers
